patterns/ml-container-cache/helm.tf

```diff
@@ -6,22 +6,8 @@ resource "helm_release" "nvidia_device_plugin" {
   name             = "nvidia-device-plugin"
   repository       = "https://nvidia.github.io/k8s-device-plugin"
   chart            = "nvidia-device-plugin"
-  version          = "0.16.2" # Matches image that is cached
+  version          = "0.17.1" # Matches image that is cached
   namespace        = "nvidia-device-plugin"
   create_namespace = true
   wait             = false
-
-  values = [
```

Comment:
this is already present on device plugin by default https://github.com/NVIDIA/k8s-device-plugin/blob/1b1ddb7e39d1ce13561b87cd848132cfe45d2c7e/deployments/helm/nvidia-device-plugin/values.yaml#L79-L84